packages/core/src/misc.ts

```diff
@@ -262,7 +267,7 @@ export function patch(newData: LooseObject, priorData: LooseObject): LooseObject
 }
 
 // This is a new, unstable simplified Patch Function to allow patching of Array Differences
-export function patchArray(newData: Array<any>, priorData: Array<any>): LooseObject {
+export function patchArray(newData: LooseObject, priorData: LooseObject): LooseObject {
```

Comment:
should this be `LooseObject|Array<any>` ?

Reply:
I debated on that for a while.  I think I will allow both.